apps/admin/backend/src/tabulation/full_results.ts

```diff
@@ -83,11 +84,13 @@ export function tabulateElectionResults({
     )) {
       const writeInSummary = groupedWriteInSummaries[groupKey];
       if (writeInSummary) {
-        groupedElectionResults[groupKey] =
-          modifyElectionResultsWithWriteInSummary(
+        groupedElectionResults[groupKey] = {
+          ...electionResults, // maintain group specifier
+          ...modifyElectionResultsWithWriteInSummary(
```

Comment:
In retrospect, I think it would be better to have two interchangeable formats for grouped election results - one that's a dictionary with keys and one that's an array with "group specifiers." Making them interchangeable would require changing the group key format to be invertible. Not prioritizing this refactor now but perhaps later